### ISSUE TITLE
[Bug]: Fenix killed due to excessive CPU, not logged in Fenix crash reporter

### ISSUE THREAD

**Issue by Cheap-Skate** [mozilla-mobile/fenix#23140]
### Steps to reproduce

I was investigating #12731, I had 40 tabs open. While the screen was off & phone in my pocket Fenix Nightly crashed completely, no trace of it and no crash report.  It as using PIDs 10909, 14008 and 13672 so given those PIDs and the time window in which it crashed I think this is relevant:-

`01-10 15:47:31.514  1392  1432 I ActivityManager: Killing 10909:org.mozilla.fenix/u0a275 (adj 850): excessive cpu 202770 during 300060 dur=249054 limit=25`
`01-10 15:47:31.740  1392  1435 E libprocessgroup: Failed to kill process cgroup uid 10275 pid 10909 in 215ms, 1 processes remain`
`01-10 15:47:31.804 14008 14031 I Gecko   : Exiting due to channel error.`
`01-10 15:47:31.806 13672 13696 I Gecko   : Exiting due to channel error.`
`01-10 15:47:32.018   751   751 I Zygote  : Process 10909 exited due to signal 9 (Killed)`
`01-10 15:47:32.031  1392  1421 W ActivityManager: setHasOverlayUi called on unknown pid: 10909`
`01-10 15:47:32.034  1392  1419 E Native_CFMS: nativePerfMgrRemoveTask    Fail    tid : 10909    return : -1`
`01-10 15:47:32.036  1392  2061 W ProcessCpusetController: Slowdown: pid : 10909 abnormal=true kill=false stay=false release=true`
`01-10 15:47:32.038  1392  2061 W ProcessCpusetController: Slowdown: pid 10909 is nonexistent`
`01-10 15:47:32.235  1392  1841 I ActivityManager: Process org.mozilla.fenix:tab5 (pid 14008) has died: fg  SVC (1474,966)`
`01-10 15:47:32.236   751   751 I Zygote  : Process 13672 exited cleanly (0)`
`01-10 15:47:32.236   751   751 I Zygote  : Process 14008 exited cleanly (0)`
`01-10 15:47:32.236  1392  1435 I libprocessgroup: Successfully killed process cgroup uid 10275 pid 14008 in 0ms`
`01-10 15:47:32.250  1392  2227 I ActivityManager: Process org.mozilla.fenix:tab27 (pid 13672) has died: fg  SVC (1474,966)`
`01-10 15:47:32.250  1392  1435 I libprocessgroup: Successfully killed process cgroup uid 10275 pid 13672 in 0ms`

### Expected behaviour

No crash or crash with lof

### Actual behaviour

Crash!

### Device name

Galaxy M32

### Android version

Android 11 

### Firefox release type

Firefox Nightly

### Firefox version

2015854827 2022-01-01

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-19108)

**Comment by Cheap-Skate** [comment]
Here it is again, phone was asleep and in my pocket, PIDs were 22207, 26278, 24683. One of the content processes got killed so I had to reload half my tabs. Nothing interesting in the log before 18:01:25.495.

01-10 18:01:25.495  1392  1432 I ActivityManager: Killing 26278:org.mozilla.fenix:tab36/u0a275 (adj 920): excessive cpu 7690 during 300150 dur=1166352 limit=2
01-10 18:01:25.714  1392  1435 E libprocessgroup: Failed to kill process cgroup uid 10275 pid 26278 in 216ms, 1 processes remain
01-10 18:01:25.756   751   751 I Zygote  : Process 26278 exited due to signal 9 (Killed)
01-10 18:01:25.758  1392  1978 W ActivityManager: Scheduling restart of crashed service org.mozilla.fenix/org.mozilla.gecko.process.GeckoChildProcessServices$tab36 in 1000ms for connection
01-10 18:01:25.762  1392  3779 W ProcessCpusetController: Slowdown: pid : 26278 abnormal=true kill=false stay=false release=true
01-10 18:01:25.762 22207 24725 I Gecko   : ###!!! [Parent][MessageChannel] Error: (msgtype=0x880008,name=PSharedWorker::Msg_Terminate) Channel error: cannot send/recv
01-10 18:01:25.762 22207 24725 I Gecko   : 
01-10 18:01:25.763 22207 24725 I Gecko   : ###!!! [Parent][MessageChannel] Error: (msgtype=0x880008,name=PSharedWorker::Msg_Terminate) Channel error: cannot send/recv
01-10 18:01:25.763 22207 24725 I Gecko   : 
01-10 18:01:25.764  1392  3779 W ProcessCpusetController: Slowdown: pid 26278 is nonexistent
01-10 18:01:25.940   594   594 I io_stats: !@ 179,0 r 1741341 33148614 w 491160 7959552 d 93007 6079356 f 253468 141202 iot 1019652 983007 th 51200 0 0 pt 0 inp 0 0 37245.558
01-10 18:01:25.983   891  1106 D connsyslogger: SD_LOG_ROOT is /data/debuglogger/connsyslog/fw.
01-10 18:01:25.983   891  1106 D connsyslogger: systemReady:1
01-10 18:01:26.094 22207 24668 I Gecko   : [Parent 22207, Unnamed thread 7d8af37340] WARNING: waitpid failed pid:26278 errno:10: file /builds/worker/checkouts/gecko/ipc/chromium/src/base/process_util_posix.cc:244
01-10 18:01:26.117 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.165  1793  1974 D NetworkController.SecMobileSignalController(0/1): onSignalStrengthsChanged level=3
01-10 18:01:26.165  1793  1974 D MobileSignalIconsSelector: getDataIconGroup(): 13
01-10 18:01:26.184 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.229 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.233 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.236 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.241 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.272 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.365 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.391   891  1107 D connsyslogger: data recycle:time out
01-10 18:01:26.391   891  1107 E connsyslogger: Bootup Log: 3 log files! 
01-10 18:01:26.428 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.474 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.521 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.567 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.627  1392  1692 I ChimeraAggressivePolicyHandler: executePolicy() - triggerSource: TRIGGER_SOURCE_LMKD
01-10 18:01:26.628  1392  1692 I ChimeraAggressivePolicyHandler: available memory: 1849244, free memory target: 1126400, quit chimera
01-10 18:01:26.642 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.750 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.830 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.836  1392  1618 E WifiVendorHal: getWifiLinkLayerStats_1_3_Internal(l.974) failed {.code = ERROR_NOT_SUPPORTED, .description = }
01-10 18:01:26.854   822   875 I BufferQueueProducer: [StatusBar$_1793#0](this:0xb400007df7af0658,id:5,api:1,p:1793,c:822) queueBuffer: fps=0.33 dur=2997.50 max=2997.50 min=2997.50
01-10 18:01:26.855   822   822 I SurfaceFlinger: SFWD update time=37246473106296
01-10 18:01:26.862   891   891 D connsyslogger: Open FIFO for close
01-10 18:01:26.863   765   765 I hwcomposer: [HWCDisplay] [Display_0 (type:1)] fps:0.334852,dur:2986.40,max:2986.40,min:2986.40  
01-10 18:01:26.880 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.911 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.948 22207 22207 W GeckoSession: Attempted to close a GeckoSession that was already closed.
01-10 18:01:26.965  1392  1392 E Native_CFMS: nativePerfMgrRemoveTask    Fail    tid : -1    return : -1

**Comment by Cheap-Skate** [comment]
Again

20 tabs open, Nightly 2022-01-11, phone had screen off and was in my pocket, Fenix was in the background.

I think Fenix PIDs were 22285 (main), 26711 (content), 26244 (content). I put a couple of log items before the critical `01-11 10:25:59.258  1392  1432 I ActivityManager` log item

I think this happens quite often, a few times a day for me. It explains why often in the morning some or all of my tabs reload, Fenix processes got killed overnight due to excessive CPU.  I have been blaming Fenix’s excessive RAM causing a low memory process kill, but I think at least some of the process kills are due to excessive cpu.

So for me 
* when Fenix is in the foreground its processes often get killed due to excessive RAM use
* when Fenix is in the background its processes sometimes get killed due to excessive CPU use

*facepalm*





01-11 10:25:59.180  1392  1452 D MARsPolicyManager: triggerAction called!
01-11 10:25:59.183  1392  1452 I LocationManagerService: getGPSUsingApps() called
01-11 10:25:59.183  1392  1452 I NSLocationMonitor: getGPSUsingApps() called
01-11 10:25:59.258  1392  1432 I ActivityManager: Killing 26244:org.mozilla.fenix:tab5/u0a275 (adj 910): excessive cpu 11200 during 300152 dur=960084 limit=2
01-11 10:25:59.260  1392  1432 I ActivityManager: Killing 26711:org.mozilla.fenix:tab2/u0a275 (adj 910): excessive cpu 14840 during 300152 dur=960153 limit=2
01-11 10:25:59.324   891  1106 D connsyslogger: SD_LOG_ROOT is /data/debuglogger/connsyslog/fw.
01-11 10:25:59.324   891  1106 D connsyslogger: systemReady:1
01-11 10:25:59.331  2812  2831 I NSMonitorService_FLP: getGPSUsingApps, NO_FREEZE={5013}/ FREEZE={10276}
01-11 10:25:59.335  1392  1452 D MARs:TopPackageFilter: filtering was skipped by screen off! com.samsung.android.calendar(0)
01-11 10:25:59.337  1392  1452 D MARs:TopPackageFilter: filtering was skipped by screen off! de.mdiener.rain.usa.plus(0)
01-11 10:25:59.339  1392  1452 D MARs:TopPackageFilter: filtering was skipped by screen off! au.com.shiftyjelly.pocketcasts(0)
01-11 10:25:59.340  1392  1452 D MARs:TopPackageFilter: filtering was skipped by screen off! com.android.vending(0)
01-11 10:25:59.341  1392  1452 D MARs:TopPackageFilter: filtering was skipped by screen off! com.sec.android.gallery3d(0)
01-11 10:25:59.342  1392  1452 D MARs:TopPackageFilter: filtering was skipped by screen off! org.mozilla.fenix(0)
01-11 10:25:59.343  1392  1452 D MARs:TopPackageFilter: filtering was skipped by screen off! com.onjara.weatherforecastuk(0)
01-11 10:25:59.345  1392  1452 D MARs:TopPackageFilter: filtering was skipped by screen off! com.channel4.ondemand(0)
01-11 10:25:59.346  1392  1452 D MARs:TopPackageFilter: filtering was skipped by screen off! com.faultexception.reader(0)
01-11 10:25:59.347  1392  1452 D MARs:TopPackageFilter: filtering was skipped by screen off! com.spencerstudios.ramlogger(0)
01-11 10:25:59.348  1392  1452 D MARs:TopPackageFilter: filtering was skipped by screen off! com.sec.android.app.camera(0)
01-11 10:25:59.368   557  3514 I android.system.suspend@1.0-service: !@SystemSuspend: wakeup_count-
01-11 10:25:59.368   557  3514 I android.system.suspend@1.0-service: !@SystemSuspend: mSuspendCounter+
01-11 10:25:59.368   557  3514 I android.system.suspend@1.0-service: !@SystemSuspend: mSuspendCounter-
01-11 10:25:59.368   557  3514 I android.system.suspend@1.0-service: !@SystemSuspend: write mem+
01-11 10:25:59.596   856   856 E ccci_rpcd: Failed to read from RPC device (-1) !! errno = 4
01-11 10:25:59.598  1392  1435 I libprocessgroup: Successfully killed process cgroup uid 10275 pid 26244 in 337ms
01-11 10:25:59.603   751   751 I Zygote  : Process 26244 exited due to signal 9 (Killed)
01-11 10:25:59.603   751   751 I Zygote  : Process 26711 exited due to signal 9 (Killed)
01-11 10:25:59.605  1392  1435 I libprocessgroup: Successfully killed process cgroup uid 10275 pid 26711 in 0ms
01-11 10:25:59.606   557  3514 I android.system.suspend@1.0-service: !@SystemSuspend: write mem-
01-11 10:25:59.606  1392  1726 I BatteryStatsService: In wakeup_callback: suspend aborted
01-11 10:25:59.607  1392  1590 W ActivityManager: Scheduling restart of crashed service org.mozilla.fenix/org.mozilla.gecko.process.GeckoChildProcessServices$tab2 in 1000ms for connection
01-11 10:25:59.621  1392  2106 W ActivityManager: Scheduling restart of crashed service org.mozilla.fenix/org.mozilla.gecko.process.GeckoChildProcessServices$tab5 in 10985ms for connection
01-11 10:25:59.638   890 22090 I CameraService: notifyMonitoredUids
01-11 10:25:59.646   890 22090 I CameraService: notifyMonitoredUids
01-11 10:25:59.659 22285 26366 I Gecko   : ###!!! [Parent][PBackgroundParent] Error: Send(msgname=PSharedWorker::Msg_Terminate) Channel error: cannot send/recv
01-11 10:25:59.659 22285 26366 I Gecko   : 
01-11 10:25:59.660 22285 26366 I Gecko   : ###!!! [Parent][PBackgroundParent] Error: Send(msgname=PSharedWorker::Msg_Terminate) Channel error: cannot send/recv
01-11 10:25:59.660 22285 26366 I Gecko   : 
01-11 10:25:59.660 22285 26366 I Gecko   : ###!!! [Parent][PBackgroundParent] Error: Send(msgname=PSharedWorker::Msg_Terminate) Channel error: cannot send/recv
01-11 10:25:59.660 22285 26366 I Gecko   : 
01-11 10:25:59.662 22285 26417 D libMEOW : applied 1 plugins for [org.mozilla.fenix]:
01-11 10:25:59.662 22285 26417 D libMEOW :   plugin 1: [libMEOW_gift.so]:
01-11 10:25:59.668 22285 26417 W libEGL  : EGLNativeWindowType 0xb400007a5af79010 disconnect failed
01-11 10:25:59.807   557  3514 I android.system.suspend@1.0-service: !@SystemSuspend: wakeup_count+
01-11 10:25:59.816 22285 26124 I Gecko   : ###!!! [Parent][PContentParent] Error: Send(msgname=PContentPermissionRequest::Msg___delete__) Channel error: cannot send/recv
01-11 10:25:59.816 22285 26124 I Gecko   : 
01-11 10:25:59.818 22285 26124 I Gecko   : ###!!! [Parent][PContentParent] Error: Send(msgname=PContentPermissionRequest::Msg___delete__) Channel error: cannot send/recv
01-11 10:25:59.818 22285 26124 I Gecko   : 
01-11 10:25:59.826  1392  1692 I ChimeraPolicyHandler: prepareForTrigger() - skip reason: screen off
01-11 10:25:59.826  1392  1692 I ChimeraAggressivePolicyHandler: executePolicy() - prepareForTrigger fails
01-11 10:25:59.988   891   891 D connsyslogger: Open FIFO for close

**Comment by Cheap-Skate** [comment]
I thought it might be add-ons, it's not. I disabled them all and it still happens. 

Here main process is PID 22285, content process PID 29046 gets killed at 15:57:39.626 for excessive CPU use. But first there are slowdown warnings.  Also, the kill seemed to happen when I walked back into coverage of my WiFi network, the WiFi stack is very chatty just before the kill. Coincidence? Dunno?

There are repeated warnings from `ProcessCpusetController` about pid 29046 slowdown, over several hours. No other process gets warnings during this time. Not sure if that's relevant

`./adb logcat | grep "ProcessCpusetController"`

01-11 13:08:01.705  1392  3774 W ProcessCpusetController: Slowdown: pid : 29046 abnormal=true kill=false stay=false release=true
01-11 13:13:57.062  1392  3772 W ProcessCpusetController: Slowdown: pid : 29046 abnormal=true kill=false stay=false release=true
01-11 13:26:33.322  1392  1726 W ProcessCpusetController: Slowdown: pid : 29046 abnormal=true kill=false stay=false release=true
01-11 13:56:35.200  1392  3774 W ProcessCpusetController: Slowdown: pid : 29046 abnormal=true kill=false stay=false release=true
01-11 14:30:12.681  1392  2061 W ProcessCpusetController: Slowdown: pid : 29046 abnormal=true kill=false stay=false release=true
01-11 14:55:59.000  1392  3764 W ProcessCpusetController: Slowdown: pid : 29046 abnormal=true kill=false stay=false release=true
01-11 15:23:17.715  1392  3674 W ProcessCpusetController: Slowdown: pid : 29046 abnormal=true kill=false stay=false release=true
01-11 15:40:36.356  1392  3759 W ProcessCpusetController: Slowdown: pid : 29046 abnormal=true kill=false stay=false release=true
01-11 15:57:39.895  1392  3700 W ProcessCpusetController: Slowdown: pid : 29046 abnormal=true kill=false stay=false release=true

Then the kill happens 15:57:39.626 just before the final `ProcessCpusetController: Slowdown` warning:-

01-11 15:57:39.521  1392  1671 D SemRoamingAssistant2.0:  mState is changed [ CONNECTED > ROAM ]
01-11 15:57:39.522  1392  1680 D WifiConnectivityMonitor: Roaming completed. Wait for 5 seconds for DHCP to begin, or not begin
01-11 15:57:39.522  1392  1671 D SemWifiScanPool: setCurrentBssid: 6c:70:9f:da:a7:d9
**01-11 15:57:39.626  1392  1432 I ActivityManager: Killing 29046:org.mozilla.fenix:tab30/u0a275 (adj 900): excessive cpu 6170 during 300163 dur=918349 limit=2**
01-11 15:57:39.629 16340 27390 I SamsungCalendar$CalendarDrawerViewImpl: init() false
01-11 15:57:39.632 16340 19828 I SamsungCalendar$SocialApis: onConnectionSuccess
01-11 15:57:39.633  1392  1618 D WifiClientModeImpl: l2Information=L2Key: W3ddf72ac4f480ce7, Cluster: G7c23033e69977d01, bssid: 00:00:00:00:00:00
01-11 15:57:39.633 28407 28935 I SEMS:Social_12.2.1.11: [GLog][Binder:28407_6][MobileServiceGroupImpl] start getGroupJoinedList
01-11 15:57:39.634 28407 28935 I SEMS:Social_12.2.1.11: [GLog][Binder:28407_6][GetGroupUseCase] UseCase execute getAllGroups
01-11 15:57:39.648  1392  1658 I WifiConnectivityMonitor.NetworkCallbackController: sNetworkCallback onCapabilitiesChanged: 630[ Transports: WIFI Capabilities: NOT_METERED&INTERNET&NOT_RESTRICTED&TRUSTED&NOT_VPN&VALIDATED&NOT_ROAMING&FOREGROUND&NOT_CONGESTED&NOT_SUSPENDED LinkUpBandwidth>=72000Kbps LinkDnBandwidth>=72000Kbps SignalStrength: -92 OwnerUid: 1000 AdministratorUids: [] SSID: "Mark and Laurie's Wi-Fi Network" RequestorUid: -1 RequestorPackageName: null]
01-11 15:57:39.657  1392  1618 D WifiClientModeImpl: l2Information=L2Key: W5c45eae43fd6cf47, Cluster: G7c23033e69977d01, bssid: 6c:70:9f:da:a7:d8
01-11 15:57:39.660 13742 13742 I wpa_supplicant: wpa_driver_nl80211_driver_cmd: wlan0 recv cmd MACADDR
01-11 15:57:39.662  1392  1618 D WifiClientModeImpl: startToCheckRoamDhcp
01-11 15:57:39.673  1392  1658 D SemWifiSwitchForIndividualAppsService: stopTCPMonitoring : already stopped
01-11 15:57:39.682 19866 19885 I Reminder$ReminderProvider: query uri [content://com.samsung.android.app.reminder/open/space_category] from com.samsung.android.calendar
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController: mWifiConfig = ID: 4 SSID: "Mark and Laurie's Wi-Fi Network" PROVIDER-NAME: null BSSID: null FQDN: null HOME-PROVIDER-NETWORK: false PRIO: 0 HIDDEN: false PMF: falseCarrierId: -1
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController:  NetworkSelectionStatus NETWORK_SELECTION_ENABLED
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController:  hasEverConnected: true
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController:  numAssociation 14
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController: update millis:1637597455457
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController: creation millis:1637597455457
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController:  validatedInternetAccess trusted
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController:  macRandomizationSetting: 1
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController:  mRandomizedMacAddress: ae:d0:2d:0f:9a:99
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController:  randomizedMacExpirationTimeMs: 01-12 15:57:37.429
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController:  KeyMgmt: WPA_PSK Protocols: WPA RSN
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController:  AuthAlgorithms: OPEN
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController:  PairwiseCiphers: TKIP CCMP
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController:  GroupCiphers: WEP40 WEP104 TKIP CCMP
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController:  GroupMgmtCiphers: BIP_GMAC_256
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController:  SuiteBCiphers:
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController:  PSK/SAE: *
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController: Enterprise config:
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController:  ocsp: 0
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController: IP config:
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController: IP assignment: DHCP
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController: Proxy settings: NONE
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController:  cuid=1000 cname=android.uid.system:1000 luid=1000 lname=android.uid.system:1000 lcuid=10064 allowAutojoin=true noInternetAccessExpected=false mostRecentlyConnected=false 
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController: recentFailure: Association Rejection code: 0
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController: wifi6Mode: false wifi6EMode: false
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController: updateWifiInfo = SSID: Mark and Laurie's Wi-Fi Network, BSSID: 6c:70:9f:da:a7:d8, MAC: 02:00:00:00:00:00, Supplicant state: COMPLETED, Wi-Fi standard: 4, RSSI: -92, Link speed: 292Mbps, Tx Link speed: 292Mbps, Max Supported Tx Link speed: 72Mbps, Rx Link speed: -1Mbps, Max Supported Rx Link speed: 72Mbps, Frequency: 2462MHz, Net ID: 4, Metered hint: false, score: 60
01-11 15:57:39.683  1793  1974 D NetworkController.WifiSignalController: SUPPLICANT_STATE_CHANGED_ACTION completed received
01-11 15:57:39.719 16340 16340 I SamsungCalendar$CalendarDrawerPresenter: setDrawerView, taskEnabled = false
01-11 15:57:39.726 28407 28421 W d.mobileservic: Reducing the number of considered missed Gc histogram windows from 115 to 100
01-11 15:57:39.849  1392  1435 E libprocessgroup: Failed to kill process cgroup uid 10275 pid 29046 in 222ms, 1 processes remain
**01-11 15:57:39.866   751   751 I Zygote  : Process 29046 exited due to signal 9 (Killed)**
01-11 15:57:39.869  1392  1408 W ActivityManager: Scheduling restart of crashed service org.mozilla.fenix/org.mozilla.gecko.process.GeckoChildProcessServices$tab30 in 1000ms for connection
01-11 15:57:39.878 22285 26366 I Gecko   : ###!!! [Parent][PBackgroundParent] Error: Send(msgname=PSharedWorker::Msg_Terminate) Channel error: cannot send/recv
01-11 15:57:39.878 22285 26366 I Gecko   : 
01-11 15:57:39.879 22285 26366 I Gecko   : ###!!! [Parent][PBackgroundParent] Error: Send(msgname=PSharedWorker::Msg_Terminate) Channel error: cannot send/recv
01-11 15:57:39.879 22285 26366 I Gecko   :
